fip-0026.md

```diff
@@ -625,20 +759,23 @@ This action facilitates transferring the domain for sale to the holder account a
 #### Exception handling
 |Error condition|Trigger|Type|fields:name|fields:value|Error message|
 |---|---|---|---|---|---|
-|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400| |"Marketplace not found"|
-|Invalid FIO domain|Supplied `fio_domain` is not listed for sale.|400| | |"FIO domain not found"|
+|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400|"marketplace"|Value sent in, e.g. "notamarketplace"|"Marketplace not found"|
 |Insufficient Funds|`buyer` doesn't have enough funds|400| | |"Not enough FIO"|
+|Invalid FIO Domain format|FIO Domain format is not valid|400|"fio_domain"|Value sent in, e.g. "alice"|"Invalid FIO domain"|
+|FIO Domain expired|FIO Domain is expired|400|"fio_domain"|Value sent in, e.g. "alice"|"FIO Domain expired. Renew first."|
+|FIO Domain not registered|FIO Domain is not registered|400|"fio_domain"|Value sent in, e.g. "alice"|"FIO Domain not registered"|
+|Not owner of FIO Domain|The signer does not own the domain|403||||
 |Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
 
 #### Response
-TBD
+none, inline action
 
 ##### Example
-TBD
+none
 
 # Rationale
 * We may not need api end-points for our actions so none are included in this document, as of right now
-* Fees are still being discussed, so those are not included either.  We would love your input on our fee structure in the second discussion link below.
+* **Domain Transfers** - We feel that having the ownership of the domain transferred to a holder account is necessary because it makes sure that when a buyer comes along that it is 100% available for sale. If the ownership doesn't transfer then the seller can still use the domain until it sells, and then, if a buyer comes along it might suddenly, without notice, disappear from their account.
```

Comment:
I assumed that the domain remains usable when listed. Is that not the case? If so many error conditions are missing, like error on register_fio_address on that domain. This needs to be clarified.